packages/sync/legacy/class.jetpack-sync-settings.php

```diff
@@ -60,7 +58,7 @@ static function get_setting( $setting ) {
 				$value = get_site_option( self::SETTINGS_OPTION_PREFIX . $setting );
 			} else {
 				// On single sites just return the default setting
-				$value = Jetpack_Sync_Defaults::get_default_setting( $setting );
+				$value                            = Jetpack_Sync_Defaults::get_default_setting( $setting );
```

Comment:
I would just add a newline below here and not bother too much about aligning the equals sign. (I guess that's the same as reverting the change?

```suggestion
				$value = Jetpack_Sync_Defaults::get_default_setting( $setting );

```

Reply:
This was done automatically by PHPCS fixer, so let's keep it 🤷‍♂️

Reply:
Automatic fixes my phpcs don't always make sense and we need to look at the code as humans as well. That's what linting is for.

Reply:
Okay, but I suggest let's fix the source of the problem though. If we don't like how phpcs fixer does it, let's make sure it does it properly. This change will come in anyway from anyone who touches that file and has phpcsfixer setup locally.

Also, I agree with the change. See the next line of code - the equal sign aligns with it after the change.